telegram/_files/inputprofilephoto.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2025
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains an objects that represents a InputProfilePhoto and subclasses."""
+
+import datetime as dtm
+from typing import TYPE_CHECKING, Optional, Union
+
+from telegram import constants
+from telegram._telegramobject import TelegramObject
+from telegram._utils import enum
+from telegram._utils.files import parse_file_input
+from telegram._utils.types import FileInput, JSONDict
+
+if TYPE_CHECKING:
+    from telegram import InputFile
+
+
+class InputProfilePhoto(TelegramObject):
+    """This object describes a profile photo to set. Currently, it can be one of
+
+    * :class:`InputProfilePhotoStatic`
+    * :class:`InputProfilePhotoAnimated`
+
+    .. versionadded:: NEXT.VERSION
+
+    Args:
+        type (:obj:`str`): Type of the profile photo.
+
+    Attributes:
+        type (:obj:`str`): Type of the profile photo.
+
+    """
+
+    __slots__ = ("type",)
+
+    def __init__(
+        self,
+        type: str,  # pylint: disable=redefined-builtin
+        *,
+        api_kwargs: Optional[JSONDict] = None,
+    ):
+        super().__init__(api_kwargs=api_kwargs)
+        self.type: str = enum.get_member(constants.InputMediaType, type, type)
+
+        self._freeze()
+
+
+class InputProfilePhotoStatic(InputProfilePhoto):
+    """A static profile photo in the .JPG format.
+
+    .. versionadded:: NEXT.VERSION
+
+    Args:
+        photo (:term:`file object` | :class:`~telegram.InputFile` | :obj:`bytes` | \
+            :class:`pathlib.Path` |): The static profile photo. |uploadinputnopath|
+
+    Attributes:
+        type (:obj:`str`): :tg-const:`telegram.constants.InputMediaType.STATIC`.
```

Comment:
```suggestion
        type (:obj:`str`): :tg-const:`telegram.constants.InputProfilePhotoType.STATIC`.
```
that or, 
```suggestion
        type (:obj:`str`): :tg-const:`telegram.constants.InputProfilePhoto.STATIC`.
```
after adding the shortcuts